### PR TITLE
fix: optimizer not working for some 3D models

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
Tested 2 common use cases of the optimizer
```
node ./index.js -i ./model.glb -m -tm 2048 -tc
node ./index.js -i ./model.glb -m -tm 2048 --force-jpg --webp -t0 256
```

### CODE REVIEW

**Review by qpitlove — APPROVED** [review]

File: packages/view3d-optimizer/index.js [hunk 1]
```diff
@@ -33,93 +32,94 @@ program.parse(process.argv);
...
+    process.stdout.cursorTo(0);
+    process.stdout.write(`- ${taskDesc}... ${chalk.green("DONE!")} (${chalk.blue(`${((endTime - startTime) / 1000).toFixed(2)}s`)})\n`)
   }
+}
 
-  if (options.basisu) {
-    await run(`npx gltf-transform etc1s ${tmpFile} ${tmpFile}`, "Applying BasisU (ETC1S) texture compression");
-  }
+const main = async () => {
+  await exec(`npx gltfpack -i ${options.input} -o ${tmpFile}`);
```
Comment (qpitlove) [inline packages/view3d-optimizer/index.js:66]: When installing a module with `npx` for the first time, it goes through a confirmation process in the terminal, but if you use the `npx --yes` option, it will be installed as a slient and usability will be improved.

Comment (WoodNeck) [reply packages/view3d-optimizer/index.js:66]: Modules will be installed as a dependency when installing view3d-optimizer, I think it doesn't install again after calling `npx`

File: packages/view3d-optimizer/index.js [hunk 2]
```diff
@@ -33,93 +32,94 @@ program.parse(process.argv);
...
-    await Promise.all(tasks).catch(err => {
-      process.stdout.write("\n");
-      err.stdout && console.error(chalk.red(err.stdout));
-      err.stderr && console.error(chalk.red(err.stderr));
-      process.exit(1);
-    });
-    const endTime = performance.now();
+const modelName = path.basename(options.input, path.extname(options.input));
+const cwd = process.cwd();
+const tmpFile = path.resolve(cwd, "temp.glb");
```
Comment (daybrush) [inline packages/view3d-optimizer/index.js:37]: Is this PR related to tmp?


The tmp-promise is missing from the code, but it seems to be in the package.json.


The module specified the tmp file directly, and is there any way to remove it again?

**Review by daybrush — APPROVED** [review]
LGTM

File: packages/view3d-optimizer/index.js (hunk 2 above)
Comment (WoodNeck) [reply packages/view3d-optimizer/index.js:37]: You're right, I was just about to do that with the other fix :)

**Review by malangfox — APPROVED** [review]
LGTM
